src/test/java/com/rocketden/main/service/SubmitServiceTests.java

```diff
@@ -66,6 +72,9 @@
     @InjectMocks
     private SubmitService submitService;
 
+    @Captor
+    ArgumentCaptor<TesterRequest> captor;
```

Comment:
This captures the argument used to call a certain method, so that way you can verify that a method is being called with the correct values (in this case, a request with null values set for non-required fields)